packages/lit-dev-content/site/tutorials/content/working-with-lists/03.md

```diff
@@ -0,0 +1,70 @@
+The rendering methods shown so far make it so that Lit keeps track of the
+DOM elements and will update them accordingly when the property items change.
+This is usually fine and efficient but sometimes it's preferable to control how
```

Comment:
I think this section needs more careful treatment. In particular, the code shown here demonstrates a BUG and should be called out as such. 

We need to clearly associate this with state being stored in the DOM that's not managed by Lit, as is shown here.

My concern is that otherwise, this seems pretty scary and like a footgun.

Reply:
^^ We explain this in the docs directly and you call it out in the next slide, but I think it should be mentioned right away up front. Otherwise, again, it seems scary.

Reply:
The first paragraph was reworded based on Kevin's earlier suggestion and I also expanded the explanation in the following paragraph that explains the current buggy behavior of the example mentioning that the checkbox state lives with the DOM elements that didn't move.